### ISSUE TITLE
mocha.parallel is printing an object after each test

### ISSUE THREAD

**Issue by fabiob** [danielstjules/mocha.parallel#38]
When using `mocha.parallel`, an object is printed after each test run. Downgrading to 0.15.3 fixes the issue.

I've created a repository with a reproducible test: https://github.com/fabiob/mocha.parallel-test-weird-print

Here's the result:

```
$ mocha


  sample test
    without mocha.parallel
      ✓ should pass
    with mocha.parallel
{
  "title": "should pass",
  "body": "function () {\n        if (spec.error) throw spec.error;\n        spec.ctx.patchContext(this);\n        return spec.promise.then(function() {\n          if (spec.error) throw spec.error;\n        });\n      }",
  "async": 0,
  "sync": true,
  "timedOut": false,
  "pending": false,
  "type": "test",
  "file": "/home/fabio/Projects/OpenSource/BugReports/mocha.parallel-prints/test/parallel_spec.js",
  "parent": "#<Suite>",
  "ctx": "#<Context>"
}
      ✓ should pass


  2 passing (9ms)
```

**Comment by fedoseev-vitaliy** [comment]
i have the same issue

**Comment by zetflo** [comment]
i have the same issue
In line 75 into file lib/contextProxy.js Console.log is present

**Comment by suguru03** [comment]
@danielstjules I have the same issue too.
I can't see the `console.log` on the master branch, but `v0.15.4` on npm seems to include `console.log`. 
Could you publish a new version?

**Comment by mbrumagin** [comment]
Not a fix, but a workaround that worked for my tests: just install v0.15.3 instead

**Comment by millerick** [comment]
@danielstjules following up on this.  It looks like what is published as `v.0.15.4` does not match what is in source control.

**Comment by heisian** [comment]
same issue here..

**Comment by iMacHumphries** [comment]
+1

**Comment by Shakes03** [comment]
+1

**Comment by danielstjules** [comment]
Published 0.15.5, sorry all!
